src/Engine/TranskribusEngine.php

```diff
@@ -22,13 +22,16 @@ class TranskribusEngine extends EngineBase
      */
     public function __construct(
         TranskribusClient $transkribusClient,
+        string $accessToken,
+        string $refreshToken,
         Intuition $intuition,
         string $projectDir,
         HttpClientInterface $httpClient
     ) {
         parent::__construct($intuition, $projectDir, $httpClient);
 
         $this->transkribusClient = $transkribusClient;
+        $this->transkribusClient->setTokens($accessToken, $refreshToken);
```

Comment:
The tokens are only injected here in order to be passed through to the client — would it be better to inject them into the client instead?